src/app/profile/page.tsx

```
@@ -0,0 +1,681 @@
+                    )}
+                  </div>
+
+                  {/* Email */}
+                  <div className="mb-4">
+                    <label
+                      className={`block text-sm font-medium mb-2 ${
+                        theme === "dark" ? "text-gray-300" : "text-gray-700"
+                      }`}
+                    >
+                      {t("emailAddress")}
+                    </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)